pmm/pmm2-server-autobuild.groovy

```diff
@@ -127,6 +130,7 @@ pipeline {
                             build-server-rpm prometheus
                             build-server-rpm alertmanager
                             build-server-rpm grafana
+                            build-server-rpm kubectl
```

Comment:
```suggestion
                            build-server-rpm percona-kubectl
```

Reply:
I want it to keep it as other third-party software that we build. For example `prometheus`, RPM spec is **prometheus.spec** but it provides **percona-prometheus** package.